jsonpb/jsonpb.go

```diff
@@ -679,16 +714,15 @@ func (u *Unmarshaler) unmarshalValue(target reflect.Value, inputValue json.RawMe
 			}
 			target.Field(0).SetString(turl)
 
-			mname := turl
-			if slash := strings.LastIndex(mname, "/"); slash >= 0 {
-				mname = mname[slash+1:]
+			resolver := u.UrlResolver
+			if resolver == nil {
+				resolver = DefaultUrlResolver
```

Comment:
How about just constructing the defaultResolver here and not need a package level var?

Reply:
Refactored to just invoke a default resolve function.